content/en/docs/Advanced/revoking-secureboot-access.md

```diff
@@ -0,0 +1,300 @@
+---
+title: "Revoking secure boot access"
+linkTitle: "Revoking secure boot access"
+weight: 10
+date: 2022-06-12
+---
+
+This document describes how an administrator can prevent certain OS images from
+booting on their hardware in the context of ["Trusted Boot"]({{< relref "../architecture/trustedboot" >}}).
+
+Two different scenarios will be covered, with the process being only slightly different for each case.
```

Comment:
it talks about two scenarios but I only see one bellow

Reply:
I pushed a commit. I tried to refer to scenario 1 and only present the differences to avoid duplicating the whole thing.